cdragontoolbox/hashes.py

```diff
@@ -10,7 +10,7 @@
 import logging
 from contextlib import contextmanager
 from typing import Dict
-from xxhash import xxh64
+from xxhash import xxh64_intdigest as intdigest
```

Comment:
It would be better to keep the full `xxh64_intdigest` name, for clarity and because each hash method usually provides its own `intdigest` and it's better to avoid name conflicts.

Reply:
👍